### PR TITLE
[6x]Gpcheckcat: Removing 'acl' and 'owner' tests from default run all option

### PR DESCRIPTION
**Pull request by piyushc01**
Requirement was to remove all and owner test from default running those, and run them when user specified it with -R option.
* removing acl and owner test from default run
* Adding behave test case for confirming the skip test in log message

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/bin/gpcheckcat [hunk 1]
```diff
@@ -3131,7 +3133,10 @@ def runAllChecks(run_tests):
     '''
     for name in sorted(all_checks, key=lambda x: all_checks[x]["order"]):
         if all_checks[name]["version"] >= GV.version and name in run_tests:
-            runOneCheck(name)
+            if (not GV.opt['-R'] and  "defaultSkip" in all_checks[name] and all_checks[name]["defaultSkip"] == True):
```
Comment (yihong0618) [inline gpMgmt/bin/gpcheckcat:3136]: for pep8  if (not GV.opt['-R'] and  "defaultSkip" in all_checks[name] and all_checks[name]["defaultSkip"] == True) ->  if (not GV.opt['-R'] and  "defaultSkip" in all_checks[name] and all_checks[name]["defaultSkip"] )  maybe better?

File: gpMgmt/bin/gpcheckcat [hunk 2]
```diff
@@ -3131,7 +3133,10 @@ def runAllChecks(run_tests):
     '''
     for name in sorted(all_checks, key=lambda x: all_checks[x]["order"]):
         if all_checks[name]["version"] >= GV.version and name in run_tests:
-            runOneCheck(name)
+            if (not GV.opt['-R'] and  "defaultSkip" in all_checks[name] and all_checks[name]["defaultSkip"]):
```
Comment (jimmyyih) [inline gpMgmt/bin/gpcheckcat:3136]: Formatting: There's an extra space in `and   "defaultSkip"` after the `and`.

File: gpMgmt/bin/gpcheckcat [hunk 3]
```diff
@@ -3131,7 +3133,10 @@ def runAllChecks(run_tests):
...
+                logger.debug("Default skipping test:"+name)
```
Comment (jimmyyih) [inline gpMgmt/bin/gpcheckcat:3137]: Log formatting: There should be a space after the colon (e.g. `test: " + name`).  The expected results in the Behave test will need to be modified accordingly too.

**Review by jimmyyih — APPROVED** [review]
Make sure to squash commits before merging.  Also, seems like this is a backport from master so addressed changes in this PR probably need to go back up to master as well.

**Review by pobbatihari — APPROVED** [review]

**Pull request merged**